app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/Sample.kt

```diff
@@ -140,10 +143,14 @@ data class Sample(
 /**
  * Starts the sample activity.
  */
-fun Sample.startSample(context: Context) {
+suspend fun Sample.startSample(context: Context) {
```

Comment:
Nitpick - "Sample" in the function name is redundant, could be:
```suggestion
suspend fun Sample.start(context: Context) {
```